go.mod

```diff
@@ -1,18 +1,15 @@
 module github.com/max-rocket-internet/k8s-event-logger
 
-go 1.18
+go 1.20
 
 require (
-	k8s.io/api v0.24.1
-	k8s.io/apimachinery v0.24.1
-	k8s.io/client-go v0.24.1
+	k8s.io/api v0.24.15
+	k8s.io/apimachinery v0.24.15
+	k8s.io/client-go v0.24.15
```

Comment:
Up to k8s 1.27 now, versions are only +/- 1 compatible, or in some circumstances +/- 2. So this probably needs to be bumped at some point in the near future.